custom_components/frigate/camera.py

```diff
@@ -86,6 +86,11 @@ def __init__(
         self._url = config_entry.data[CONF_URL]
         self._attr_is_streaming = self._camera_config.get("rtmp", {}).get("enabled")
         self._attr_is_recording = self._camera_config.get("record", {}).get("enabled")
+        # Set the use_wallclock_as_timestamps stream option if HA>=2022.6
+        if MAJOR_VERSION > 2022 or (MAJOR_VERSION == 2022 and MINOR_VERSION >= 6):
```

Comment:
I'm not sure if this is needed? We have a major release for this integration and have made the minimum HA version 2022.6.1